js/index.js

```diff
@@ -1,7 +1,106 @@
 // Iteration 1: Names and Input
+let hacker1 = "Sabela";
+console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Nina";
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  } else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+  }
 
+// Iteration 3: Loops
+/* 3.1 Print all the characters of the driver's name, separated by a space and [in capitals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase)
+  i.e. `"J O H N"` */
 
-// Iteration 3: Loops
\ No newline at end of file
+hacker1 = hacker1.toUpperCase().split("").join(" "); 
+console.log(hacker1);
+
+/* 3.2 Print all the characters of the navigator's name, in reverse order. 
+  i.e. `"nhoJ"`*/
+
+for (let i = (hacker1.length); i >= 0; i-- ){
+    console.log(hacker1.charAt(i))
+}
+
+/*3.3 Depending on the [lexicographic order](https://en.wikipedia.org/wiki/Lexicographical_order) of the strings, print: <br>
+    - `The driver's name goes first.` <br>
+    - `Yo, the navigator goes first definitely.` <br>
+    - `What?! You both have the same name?`*/
+
+switch(hacker1.localeCompare(hacker2)) {
+    case 1: console.log(`The driver's name goes first.`);
```

Comment:
This output looks good. Please check the MDN documentation. "2" and "-2" are possible outcomes as well. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
Just in case I added then these extra cases